### DISCUSSION TITLE
Osu! Lazer unable to find multi lobbies

### DISCUSSION THREAD

**Discussion by dizzylad** [ppy/osu#15935] [Q&A]
My mate is able to connect, join and create multi lobbies but I'm unable to even get past the circle loading screen and no lobbies show

I've tried signing out and back in but nothing works, restarting pc and the game but nothing.

Video or screenshot showing the problem: https://imgur.com/11qqE1M

osu! version: Lazer 2021.1202.0 (latest)

**Comment by bdach** [comment]
Please attach logs from either:
* `%AppData%/osu/logs` (on Windows),
* `~/.local/share/osu/logs` (on Linux and macOS),
* `Android/Data/sh.ppy.osulazer/logs` (on Android).

**Reply by dizzylad** [reply]
hey, is this in local, locallow or roaming?

**Reply by dizzylad** [reply]
in %AppData%

**Reply by dizzylad** [reply]
https://we.tl/t-6fCFGxJH4y , these are all logs from osu lazer, database, network, performance, runtime, updater

**Comment by bdach** [comment]
please attach the logs by dragging them into the comment box rather than using external services. i cannot seem to be able to access them as you've attached them.

**Reply by dizzylad** [reply]
[database.log](https://github.com/ppy/osu/files/7654278/database.log
[network.log](https://github.com/ppy/osu/files/7654279/network.log)
[performance.log](https://github.com/ppy/osu/files/7654280/performance.log)
)
[runtime.log](https://github.com/ppy/osu/files/7654281/runtime.log)
[updater.log](https://github.com/ppy/osu/files/7654282/updater.log)

**Reply by bdach** [reply]
from the log:

```
2021-12-04 16:40:13 [verbose]: OnlineMultiplayerClient connection error: System.AggregateException: Unable to connect to the server with any of the available transports. (WebSockets failed: The server returned status code '404' when status code '101' was expected.) (ServerSentEvents failed: The transport does not support the 'Binary' transfer format.) (LongPolling failed: Response status code does not indicate success: 404 (Not Found).)
2021-12-04 16:40:13 [verbose]: ---> Microsoft.AspNetCore.Http.Connections.Client.TransportFailedException: WebSockets failed: The server returned status code '404' when status code '101' was expected.
2021-12-04 16:40:13 [verbose]: ---> System.Net.WebSockets.WebSocketException (0x80004005): The server returned status code '404' when status code '101' was expected.
2021-12-04 16:40:13 [verbose]: at System.Net.WebSockets.WebSocketHandle.ConnectAsync(Uri uri, CancellationToken cancellationToken, ClientWebSocketOptions options)
```

it looks like there's an issue with establishing a connection to multi using signalr. are you using a firewall, or using an internet connection that may be blocking websocket traffic?

**Reply by dizzylad** [reply]
hmmmm right could it be i changed my dns to 1.1.1.1?

**Reply by dizzylad** [reply]
but apart from that no, i dont use anything so im not sure

**Reply by dizzylad** [reply]
osu stable works for multi, everyother game works for multiplayer but this is the only one that isnt working, not too sure whats happened

**Reply by peppy** [reply]
This is a 404 not found error. That means somehow your routing to the server is completely incorrect. You may have done some stupid stuff to your HOSTS file or certificate setup. If so, undo that.

If that's not the case, your IP or router may be blocking the game. Try visiting https://spectator.ppy.sh and see if it downloads a response, or if you see some other error message.

**Reply by dizzylad** [reply]
Hey, I am able to download the file and also I’ve not touched anything to do with host files, just downloaded the game imported over songs and skins and tried to play with my mate

**Reply by peppy** [reply]
Do you have any other devices on the same network which you can test the same flow on, and see if you get the same issue? A good starting point would be to figure out whether the issue is at your network level, or specific to your windows install.

**Reply by dizzylad** [reply]
Okay, i could download the file on my phone but unable to download the file on the pc, not sure what could cause this


This spectator.ppy.sh page can’t be foundNo web page was found for the web address: https://spectator.ppy.sh/
HTTP ERROR 404

**Reply by peppy** [reply]
That seems like you're definitely not hitting the correct server, for some reason. If possible, can you use the web inspector in your browser and export an HAR file or similar?

**Reply by dizzylad** [reply]
Im not sure on how to do that, umm let me figure that out one sec haha

**Reply by dizzylad** [reply]
it isnt allowing me to upload the file here 

![image](https://user-images.githubusercontent.com/88896545/144782821-1f273499-fd93-4948-8b27-16d366fb8b55.png)

**Reply by dizzylad** [reply]
I opened the har file and copied and pasted into a txt doc

**Reply by peppy** [reply]
Thanks, I've retrieved the file. I should have mentioned that it may contain secure content (would recommend you end your osu! web sessions immediately as a precaution via the bottom of https://osu.ppy.sh/home/account/edit)

**Reply by peppy** [reply]
I'll look into the file content later today and report back if I find something actionable.

**Reply by dizzylad** [reply]
I ended all sessions, am I able to reconnect them? also thank you!

**Reply by peppy** [reply]
Yes, that's fine. Your previous keys were exposed in the attached file unfortunately, but your password and future sessions are safe.

**Reply by dizzylad** [reply]
Thank you, i changed password just incase since I wasn't sure

**Reply by dizzylad** [reply]
I’ve tried everything I can think of, I use eset anti virus and I turned the network protection off but still the same thing and still unable to find multi lobbies

**Reply by dizzylad** [reply]
Hey sorry for messaging this again, is there anything you found?

**Reply by peppy** [reply]
Unfortunately I couldn't find any leads yet.

**Reply by dizzylad** [reply]
No worries thank you
